packages/in_app_purchase/in_app_purchase_platform_interface/lib/src/in_app_purchase_platform_addition.dart

```diff
@@ -2,18 +2,33 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:in_app_purchase_platform_interface/in_app_purchase_platform_interface.dart';
+
 // ignore: avoid_classes_with_only_static_members
 /// The interface that platform implementations must implement when they want to
-/// provide platform specific in_app_purchase features.
+/// provide platform-specific in_app_purchase features.
+///
+/// Platforms that wants to introduce platform-specific public APIs should create
+/// a class that either extend or implements [InAppPurchasePlatformAddition]. Then replace
```

Comment:
s/replace ... with/set ... to/ (by default there's no instance, so there's nothing to replace)

Reply:
done